app/models/build.js

```diff
@@ -159,4 +163,9 @@ export default Model.extend(DurationCalculations, {
       return m.isValid() ? m.format('lll') : 'not finished yet';
     }
   }),
+
+  increasePriority(shouldCancelRunningJobs) {
+    const url = `/build/${this.id}/priority?cancel_all=${shouldCancelRunningJobs}`;
```

Comment:
I would like to suggest making this url a little bit more predictable:
```suggestion
    const url = `/build/${this.id}/priority?cancel_all=${shouldCancelRunningJobs ? true : false}`;
```
This would prevent all the `null`, `undefined`, etc. to be assigned to the URL.

Reply:
Hello, @artursmirnov it is giving _no-unneeded-ternary_ lint issue.

Reply:
Hence now implementing it with explicit if conditions.